computing-resources/neuropoly/data/git-datasets.md

```diff
@@ -83,7 +83,10 @@ You should also make sure to [configure git annex](./git-annex.md#global-git-con
 To see what datasets you have available, use `info`, for example:
 
 ```
-$ ssh git@data.neuro.polymtl.ca info
+ssh git@data.neuro.polymtl.ca info
+```
+And the output would look like this:
+```
```

Comment:
Pretty sure this needs a linebreak to parse ?

````suggestion
ssh git@data.neuro.polymtl.ca info
```

And the output would look like this:
```
````